x/delayedack/types/codec.go

```diff
@@ -28,3 +24,18 @@ func RegisterInterfaces(reg types.InterfaceRegistry) {
 	)
 	msgservice.RegisterMsgServiceDesc(reg, &_Msg_serviceDesc)
 }
+
+var (
+	Amino     = codec.NewLegacyAmino()
+	ModuleCdc = codec.NewAminoCodec(Amino)
+)
+
+func init() {
+	RegisterCodec(Amino)
```

Comment:
if this is already called in module.go, in `RegisterLegacyAminoCodec` , why do it again?